psi4/src/psi4/libtrans/integraltransform_moinfo.cc

```diff
@@ -275,7 +275,7 @@ void IntegralTransform::process_spaces() {
                         if (mosym_[orb] == h) {
                             aOrbsPI[h]++;
                             aOrbSym[count] = h;
-                            if (aIndex) aIndex[count] = aindex[n];
+                            //if (aIndex) aIndex[count] = aindex[n];
```

Comment:
I think I'm being slow here, but I don't see what the problem is.  The (poorly named) `aIndex` pointer is created if custom array indexing was provided via `aindex`.  If that vector is empty, no copy is attempted; if not, the loops should essentially reorder the `aindex` vector into `aIndex`, but sorted by irrep.  There should probably be a catch to ensure that the number of orbitals and indices is the same, but I think that exists somewhere upstream.

Reply:
Is there a particular compiler error or failed test that led to this change?

Reply:
Also, whatever was the reason for this change, why does it not come into play [for the beta orbital indexing](https://github.com/psi4/psi4/blob/ab090fd8672f36e1b48dfe1f643e623262bf9c56/psi4/src/psi4/libtrans/integraltransform_moinfo.cc#L478) where the logic is identical with different variable names?

Reply:
On Windows, the tests
- casscf-fzc-sp
- casscf-sa-sp
- casscf-semi
- casscf-sp
- rasscf-sp
- molden1
- molden2

crash with run-time `vector index error`, which happens on that line. In case of beta orbitals, maybe there are no tests for that.

So I not sure, how to make a proper fix.